sw/device/silicon_creator/manuf/lib/util.c

```diff
@@ -0,0 +1,38 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/silicon_creator/manuf/lib/util.h"
+
+#include <stdint.h>
+
+#include "sw/device/lib/base/status.h"
+#include "sw/device/lib/crypto/include/datatypes.h"
+#include "sw/device/lib/crypto/include/hash.h"
+
+OT_WARN_UNUSED_RESULT
```

Comment:
I think this `OT_WARN_UNUSED_RESULT` may be redundant.

Reply:
Oops, copy paste error, fixed